

drivers/usb/net/pegasus.c: In function `pegasus_set_settings':
drivers/usb/net/pegasus.c:961: warning: passing arg 2 of `mii_ethtool_sset' from incompatible pointer type
drivers/usb/net/pegasus.c: In function `pegasus_get_msglevel':
drivers/usb/net/pegasus.c:978: warning: unused variable `pegasus'
drivers/usb/net/pegasus.c: In function `pegasus_set_msglevel':
drivers/usb/net/pegasus.c:985: warning: unused variable `pegasus'

Signed-off-by: Andrew Morton <akpm@osdl.org>
---

 25-akpm/drivers/usb/net/pegasus.c |   14 +++++++++-----
 1 files changed, 9 insertions(+), 5 deletions(-)

diff -puN drivers/usb/net/pegasus.c~pegasus-fixes drivers/usb/net/pegasus.c
--- 25/drivers/usb/net/pegasus.c~pegasus-fixes	2004-08-30 21:56:49.079627920 -0700
+++ 25-akpm/drivers/usb/net/pegasus.c	2004-08-30 21:56:49.084627160 -0700
@@ -958,7 +958,7 @@ static int pegasus_get_settings(struct n
 static int pegasus_set_settings(struct net_device *dev, struct ethtool_cmd *ecmd)
 {
 	pegasus_t *pegasus = netdev_priv(dev);
-	return mii_ethtool_sset(&pegasus->mii, &ecmd);
+	return mii_ethtool_sset(&pegasus->mii, ecmd);
 }
 
 static int pegasus_nway_reset(struct net_device *dev)
@@ -975,15 +975,19 @@ static u32 pegasus_get_link(struct net_d
 
 static u32 pegasus_get_msglevel(struct net_device *dev)
 {
-	pegasus_t *pegasus = netdev_priv(dev);
-	/* return pegasus->msg_enable; FIXME */
+	/*
+	 * pegasus_t *pegasus = netdev_priv(dev);
+	 * return pegasus->msg_enable; FIXME
+	 */
 	return 0;
 }
 
 static void pegasus_set_msglevel(struct net_device *dev, u32 v)
 {
-	pegasus_t *pegasus = netdev_priv(dev);
-	/* pegasus->msg_enable = edata.data; FIXME */
+	/*
+	 * pegasus_t *pegasus = netdev_priv(dev);
+	 * pegasus->msg_enable = edata.data; FIXME
+	 */
 }
 
 static struct ethtool_ops ops = {
_
